### PR TITLE
add content/continuous_delivery.md

### PR DESCRIPTION
**Pull request by Submarinee**
add content/continuous_delivery.md.

### CODE REVIEW

File: content/zh/continuous_delivery.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+## 是什么
+持续交付，通常缩写为 CD，是一组实践，其中代码更改自动部署到验收环境中（或者，在持续部署的情况下，部署到生产环境中）。持续交付至关重要地包括确保在部署之前对软件进行充分测试的程序，并在认为必要时提供回滚更改的方法。 持续集成 (CI) 是实现持续交付的第一步（即，更改必须在测试和部署之前彻底合并）。
+
+## 解决的问题
+部署 [可靠](/reliability/) 更新成为一个大规模的问题。 理想情况下，我们会更频繁地部署以向最终用户提供更好的价值。 但是，手动操作会导致每次更改的交易成本很高。 从历史上看，为了避免这些成本，组织发布的频率较低，一次部署更多更改并增加出现问题的风险。
+
+## 如何帮助
+持续交付策略创建了一条完全自动化的生产路径，该路径使用各种部署策略（例如 [金丝雀](/canary_deployment/) 或 [蓝绿色](/blue_green_deployment/) 版本）测试和部署软件。 这允许开发人员频繁地部署代码，让他们放心，新版本已经过测试。 通常，基于主干的开发用于 CD 策略，而不是功能分支或拉取请求。
```
Comment (hanyuancheung) [inline content/zh/continuous_delivery.md:14]:
```suggestion
持续交付策略创建了一条完全自动化的生产路径，该路径使用各种部署策略（例如 [金丝雀发布](/canary_deployment/) 或 [蓝绿部署](/blue_green_deployment/) 版本）测试和部署软件。 这允许开发人员频繁地部署代码，让他们放心，新版本已经过测试。 通常，基于主干的开发用于 CD 策略，而不是功能分支或拉取请求。
```

**Review by hanyuancheung — APPROVED** [review]
little suggestion

**Review by Jacob953 — APPROVED** [review]
LGTM.

**Comment by iamNoah1** [comment]
looks like you need to rebase :)

**Comment by Submarinee** [comment]
> looks like you need to rebase/merge :)

Thx~ I will take your suggestion.

**Pull request closed without merging**